src/main/scala/org/globalforestwatch/layers/ForestAgeCategory.scala

```diff
@@ -2,23 +2,21 @@ package org.globalforestwatch.layers
 
 import org.globalforestwatch.grids.GridTile
 
-case class ForestAgeCategory(gridTile: GridTile) extends StringLayer with OptionalILayer {
+case class ForestAgeCategory(gridTile: GridTile, model: String = "standard")
+  extends StringLayer
+    with OptionalILayer {
+  //    val model_suffix = if (model == "standard") s"__standard" else s"__$model"
+  val model_suffix = s"__$model"
+  val uri: String =
+    s"$basePath/gfw_forest_age_category$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/category/geotiff/${gridTile.tileId}.tif"
 
-  val uri: String = s"$basePath/gfw_forest_age_category/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/category/geotiff/${gridTile.tileId}.tif"
```

Comment:
Need to change layer path in datalake to "$basePath/gfw_forest_age_category$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/category/geotiff/${gridTile.tileId}.tif"